catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/inputtransformer/InputTransformerProducer.java

```diff
@@ -117,6 +118,8 @@ protected Object transform(
                           mimeType, MimeTypeToTransformerMapper.ID_KEY, transformerId)));
     } catch (IOException e) {
       throw new CatalogTransformerException("Unable to transform incoming product", e);
+    } catch (TypeConversionException e) {
```

Comment:
I guess that's true. In that case I believe leaving the original catch would be better since the Synchronization will log a message about the disconnect

Reply:
Only concern would be if there are really busy logs then it could be a bit disjointed

Reply:
What was this catch accomplishing? It's also an unchecked exception, so we should avoid catching it if we can.